apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx

```diff
@@ -61,37 +53,23 @@ const useNoAnimationStyles = makeStyles({
     },
   },
 });
-export const TestWrapperDecoratorNoAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
-  const noAnimationStyles = useNoAnimationStyles();
-  const className = mergeClasses(noAnimationStyles.root, 'testWrapper');
-  return (
-    <div style={{ display: 'flex' }}>
-      <div
-        className={className}
-        style={{
-          padding: '10px',
-          overflow: 'hidden',
-        }}
-      >
-        {story()}
-      </div>
-    </div>
-  );
-};
 export const TestWrapperDecoratorPauseAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
-  const noAnimationStyles = useNoAnimationStyles();
+  const noAnimationStyles = usePausedAnimationStyles();
   const className = mergeClasses(noAnimationStyles.paused, 'testWrapper');
```

Comment:
sorry, one last naming nit
```suggestion
  const pausedAnimationStyles = usePausedAnimationStyles();
  const className = mergeClasses(pausedAnimationStyles.paused, 'testWrapper');
```